types/google.visualization/index.d.ts

```diff
@@ -1724,6 +1731,61 @@ declare namespace google {
             yellowTo?: number;
         }
 
+        //#endregion
+        //#region Sankey
+
+        // https://developers.google.com/chart/interactive/docs/gallery/sankey
+        export class Sankey extends ChartBaseClearable {
+            draw(data: DataTable | DataView, options: SankeyChartOptions): void;
+            getBoundingBox(id: string): ChartBoundingBox;
+        }
+
+        // https://developers.google.com/chart/interactive/docs/gallery/sankey#configuration-options
+        export interface SankeyChartOptions {
+            width?: number;
+            forceIFrame?: boolean;
+            height?: number;
+            sankey?: {
+                iterations?: number;
+                link?: {
+                    color?: {
+                        fill?: string;
+                        fillOpacity?: number;
+                        stroke?: string;
+                        strokeWidth?: number;
+                    };
+                    colorMode?: 'none' | 'source' | 'target' | 'gradient';
+                    colors?: string[];
+                };
+                node?: {
+                    colorMode?: 'unique';
+                    colors?: string[];
+                    interactivity?: boolean;
+                    label?: {
+                        bold?: boolean;
+                        color?: string;
+                        fontName?: string;
+                        fontSize?: number;
+                        italic?: boolean;
+                    };
```

Comment:
```suggestion
                    label?: ChartTextStyle;
```
You can reuse `ChartTextStyle`